### DISCUSSION TITLE
Getting error while adding slack app on SEQ server.

### DISCUSSION THREAD

**Discussion by Zaheerkhan999** [datalust/seq-tickets#1885] [Help]
Hi ,

We are getting an error while adding seq.slack.app from UI and getting the below error.

```
Could not install app
Package Seq.App.Slack could not be interrogated for app configuration: no type marked with `[SeqApp]` could be found.
```

Could anyone help in this regard?
![image (1)](https://github.com/datalust/seq-tickets/assets/55441372/a51ea1ed-0145-42db-b668-8b4efa436f4c)

Thanks,
Zaheer Khan

**Comment by liammclennan** [comment]
Hi @Zaheerkhan999 

Not all Seq versions and app versions are compatible. Try upgrading to the latest version of Seq or using a version of the Slack app released around the time of your Seq version (note that this could be missing some security patches and things).
